recipes/KsponSpeech/ksponspeech_prepare.py

```diff
@@ -3,13 +3,15 @@
 
 Author
 ------
-Dongwon Kim, Dongwoo Kim 2021
+Dong Won Kim, Dongwoo Kim 2021
+Dong Won Kim 2024
 """
 
 import csv
 import logging
 import os
 import re
+from multiprocessing import Pool, cpu_count
```

Comment:
Note: we already have a tool for multiprocessing in speechbrain: https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/utils/parallel.py#L202

You can find examples here: https://github.com/speechbrain/speechbrain/blob/develop/recipes/CommonVoice/common_voice_prepare.py#L328-L346 and https://github.com/speechbrain/speechbrain/blob/develop/recipes/LibriSpeech/librispeech_prepare.py#L329-L333

Reply:
I have just updated the script accordingly